migrations/1590811304587_drop-columns-on-audit-table.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
+  pgm.dropColumns("stakeholder_log", "verified_date", { ifExists: true });
```

Comment:
I noticed in testing the trigger that these cols were not in stakeholder table. If that's incorrect or unexpected I can remove

Reply:
You are right - neither of these cols is in the stakeholder table.  I'm going to merge and give it a go...